File: Arrays/Python/Mini-Max-Sum.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'miniMaxSum' function below.
#
# The function accepts INTEGER_ARRAY arr as parameter.
#

def miniMaxSum(arr):
    # Write your code here
    arr.sort()
    maxsum, minsum = 0,0
    
    for i in range(0,len(arr)-1):
        maxsum = maxsum + arr[i]

    for j in range(1, len(arr)):
        minsum = minsum + arr[j]
        
    print(maxsum, minsum)

if __name__ == '__main__':

    arr = list(map(int, input().rstrip().split()))

    miniMaxSum(arr)

File: Arrays/Mini-Max-Sum.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

void miniMaxSum(vector<int> arr) {
    sort(arr.begin(), arr.end());
    long long maxsum = 0, minsum = 0;

    for (int i = 0; i < arr.size() - 1; ++i) {
        maxsum += arr[i];
    }

    for (int j = 1; j < arr.size(); ++j) {
        minsum += arr[j];
    }

    cout << maxsum << " " << minsum << endl;
}

int main() {
    vector<int> arr(5);

    for (int i = 0; i < 5; ++i) {
        cin >> arr[i];
    }

    miniMaxSum(arr);

    return 0;
}
